src/FileMetadata/typings/index.ts

```diff
@@ -0,0 +1,69 @@
+import { LearningObject } from '../../shared/entity';
+export type LearningObjectFile = LearningObject.Material.File;
+
+/**
+ * An interface representation of the FileMetadataDocument type
+ *
+ * @export
+ * @interface FileMetadataDocument
+ */
+export interface FileMetadataDocument {
+  /**
+   * Unique identifier of the file document
+   */
+  id: string;
+  /**
+   * Time at which the file metadata was created (When the file was first ever uploaded)
+   * Value is a string representing the milliseconds elapsed since the UNIX epoch
+   */
+  createdDate: string;
+  /**
+   * User inputted information about the contents of the file
+   */
+  description: string;
+  /**
+   * Server generated hash of the file's content
+   */
+  ETag: string;
+  /**
+   * Suffix identifier indicating file's type or characteristics of the file's contents
+   */
+  extension: string;
+  /**
+   * The location the file was uploaded to within the file system's hierarchy
```

Comment:
I think that with the nature of the way we’re doing storage we run the risk of not being clear when we’re talking about the file system as the Learning Object understands it and the file system as we have the files placed in actual storage. What about saying this? 

```suggestion
   * The location the file was uploaded to within the Learning Object’s file system hierarchy
```